### PR TITLE
SAAP onboarding

### PR DESCRIPTION
**Pull request by karl-johan-grahn**
Onboarding process

### CODE REVIEW

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-73-951b6b22`

File: docs/content/onboarding/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Configure SSO
+
+## Week 2-4 - Application Migration - Time and Material - Optional
+
+- Select the first application
+
+- Educate team on key concepts
+
+- Containerize application if required
+
```
Comment (AsfaMumtaz) [inline docs/content/onboarding/onboarding.md:26]: We also need to create a chart after this step

File: docs/content/onboarding/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- Configure access to systems
+
+- On-boarding session and knowledge sharing
+
+- Configure SSO
+
+## Week 2-4 - Application Migration - Time and Material - Optional
+
+- Select the first application
```
Comment (mustafaStakater) [inline docs/content/onboarding/onboarding.md]: 1. Setup infra & apps gitops repository for deploying application and dependent configurations.

2. Setup the tenant operator configurations and decided on the number of environments if required.

3. Select the application or group of applications that work together and divide them into microservices  and containerize them.

4. Deploy the application onto the cluster using Kubernetes Resources. List out secret required, dependent services, databases, storage requirements and implement them accordingly.

5. Deploy Kubernetes Resources using Stakater Developed Helm Chart on the environment your choice (prod,stage,test,dev) in apps-gitops-config.
Available at  : https://github.com/stakater/application

6. Implement CI pipeline for the application if required. Deploy tekton CI pipeline using Stakater Tekton Chart.

7. Implement monitoring, logging, validation/testing frameworks or any additional tools for the application.

8. Repeat the steps for other applications 2 to 7 for other application(s).

**Review by AsfaMumtaz — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-73-2eeb0ac6`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-73-b62637d5`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-73-313af89e`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-73-ce7ee60e`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-73-2c38b863`

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-73-80e8ffc8`

**Review by AsfaMumtaz — APPROVED** [review]

**Comment by stakater-user** [comment]
@karl-johan-grahn Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:v0.0.51`

**Pull request merged**